File: app/src/main/java/org/mifos/mobile/ui/views/SavingsAccountWithdrawView.java
package org.mifos.mobile.ui.views;

/*
 * Created by saksham on 02/July/2018
 */

import org.mifos.mobile.ui.views.base.MVPView;

public interface SavingsAccountWithdrawView extends MVPView {

    void showUserInterface();
    void submitWithdrawSavingsAccount();
    void showSavingsAccountWithdrawSuccessfully();
    void showMessage(String message);
    void showError(String error);
}

File: app/src/main/java/org/mifos/mobile/ui/views/SavingsAccountWithdrawView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.ui.views.base.MVPView

/*
* Created by saksham on 02/July/2018
*/   interface SavingsAccountWithdrawView : MVPView {
    fun showUserInterface()
    fun submitWithdrawSavingsAccount()
    fun showSavingsAccountWithdrawSuccessfully()
    fun showMessage(message: String?)
    fun showError(error: String?)
}